node/resolvers/items.ts

```diff
@@ -154,6 +168,19 @@ export const mutations = {
         previousItems
       )
 
+      const updatedSubscriptionData = {
+        subscriptions: newOrderForm.subscriptionData
+          ? newOrderForm.subscriptionData.subscriptions.concat(
+              newSubscriptionDataEntries
+            )
+          : newSubscriptionDataEntries,
+      }
+
+      await checkout.updateSubscriptionDataField(
```

Comment:
It will always do this request? Shouldn't it be under a condition?

Reply:
Yep, it should be under a conditional!